### ISSUE TITLE
pylast.WSError: Authentication Failed - You do not have permissions to access the service.

### ISSUE THREAD

**Issue by yongcamu** [pylast/pylast#185]
Hi, I just started giving pylast a go but I can't authenticate myself.

I used the example provided in the pylast README. Registered myself an API key and replaced the 4 variables which I needed to but I am prompted with the error above.

If makes any difference:
- I'm testing this out locally so I registered my API callback to http:127.0.0.1.
- I can't login to [Scrobble to Lastfm] on Spotify too but logging in to Lastfm site works fine.

Thank you for your time!

**Comment by hugovk** [comment]
Which line exactly gives the error? This one?
```python
network = pylast.LastFMNetwork(api_key = API_KEY, api_secret =
    API_SECRET, username = username, password_hash = password_hash)
```
 
I've just tested, and it's working for me.

Double check you have the right values for the four variables, make sure you copied every character, and that you use the password hash and not the password itself: `password_hash = pylast.md5("your_password")`.

Also, for read-only use and debugging, you can skip the username and password:
```python
network = pylast.LastFMNetwork(api_key = API_KEY, api_secret = API_SECRET)
```

**Comment by yongcamu** [comment]
Hi hugovk,

Thank you for your reply. Yes, it was that line of code in the error message.

I found out the reason why I couldn't authenticate after re-registering a new account. My old account was created a while back, it had a pure alphabet password which doesn't adhere to their current password requirements. Despite being able to login to the site, the API must have not allowed it. I managed to authenticate after I updated my password.
```
New passwords must:
- Be at least six characters long
- Contain one or more numbers
- Include at least one of the following special characters: !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~, or a space
```

Thanks again! :D
